### DISCUSSION TITLE
[Hardware donation/loan] Support for Hampton Bay (sold at Home Depot in US) doorbells

### DISCUSSION THREAD

**Discussion by jscott0** [merbanan/rtl_433#2502] [General]
Hi,

I've fiddled with flex decoders, but I really don't understand what all the parameters mean (I'm not a radio person), but these specific doorbells are extremely common in the United States and for a personal project I'd like there to be first-class support in rtl_433. Could I loan the doorbell to somebody who could work on or research it? My doorbell has FCC ID BJ4-WLTX301-A

Thanks for your consideration

**Comment by zuckschwerdt** [comment]
If you can grab a good sample file then all parameters can be found from that. https://triq.org/rtl_433/ANALYZE.html
Just upload as zip here.

**Reply by jscott0** [reply]
Thank you for the kind advice. I've uploaded a thingaroo with the signal [here](https://github.com/merbanan/rtl_433/files/11467476/doorbell-siggy.zip)

**Comment by zuckschwerdt** [comment]
That sample file is eerily quiet. The gain on your receiver seems to be way too low. Is that a Fitipower FC0012 tuner (e.g. not a R820T) by chance? They Autogain does not work well there and you might have more luck with a manual gain like `-g 20`

Grab one of our sample files, e.g. https://github.com/merbanan/rtl_433_tests/tree/master/tests/door_bell/01
and compare on https://triq.org/pdv/
